### DISCUSSION TITLE
PGNet loss不下降以及AI studio问题

### DISCUSSION THREAD

**Discussion by forAlgorithm** [PaddlePaddle/PaddleOCR#12230] [Q&A] [answered]
PGNet loss从200下降到20以后怎么也下不去了，还会在20~50间反复很挑，用的都是默认配置，还有就是免费的V10032G可以跑我的PGNet ，为什么我充了钱换成V100 40G的一直反而提示我内存不足

<img width="847" alt="c1a234239f4137cc7c5ec502480ca57" src="https://github.com/PaddlePaddle/PaddleOCR/assets/47384518/171304a7-854d-48bb-be43-a837bc9d7af5">

**Comment by forAlgorithm** [comment]
![image](https://github.com/PaddlePaddle/PaddleOCR/assets/47384518/93709a26-e311-40c7-aa7f-0294ff143b4f)

**Comment by Stonesusu** [comment, marked as answer]
在配置文件中增加了use_shared_memory: False，解决shared_memory报错问题
实例：
  loader:
    shuffle: True
    drop_last: True
    batch_size_per_card: 8
    num_workers: 8
    use_shared_memory: False  此行为新增
